<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 255, 212);</string>
  </property>
  <widget class="QLabel" name="label_tariffPlan">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono Light</family>
     <pointsize>27</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Тарифный план</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>761</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>106</width>
     <height>122</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_open">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_insert">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>423</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_imia">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название тарифа</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_tariff">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <item>
       <property name="text">
        <string>Минимум</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выгодный</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Средний</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Интернет</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Звонки</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_imia_2">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_ctoimost">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <item>
       <property name="text">
        <string>1000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1650</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_imia_3">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Минуты</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_minut">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_imia_4">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Интернет</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_internet">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_imia_5">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Услуги</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <item>
       <property name="text">
        <string>Кто звонил</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Видеозвонки</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Безлимит на соц.сети</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Международный роуминг</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
